lib/src/text-input/Suggestion.tsx

```diff
@@ -3,13 +3,16 @@ import styled from "styled-components";
 import { SuggestionProps } from "./types";
 
 const transformSpecialChars = (str: string) => {
-  const specialCharsRegex = /[\\*()\[\]{}+?/]/;
+  const specialCharsRegex = /[\\*()[\]{}+?/]/;
```

Comment:
Accor

According to ESLint, it seems that the backslash to escape the square bracket was unneeded. I have checked it and it seems to work equally this way but I would suggest you to test it just in case.